src/com/tambolaassingment/Player.java

```diff
@@ -23,12 +24,12 @@ public Player(String name, TicketChecker ticketChecker) {
 
     }
 
-    public HashMap<Integer, Boolean> getTicket() {
+    public Map<Integer, Boolean> getTicket() {
         return ticket;
     }
 
 
-    void displayticket(HashMap<Integer, Boolean> ticket) {
+    void displayticket(Map<Integer, Boolean> ticket) {
         Iterator<Integer> ticketKey = ticket.keySet().iterator();
```

Comment:
You can change Iterator to inhenached for loop for better implementation

Reply:
done it